### ISSUE TITLE
Incorporate text to speech as an export type

### ISSUE THREAD

**Issue by beckej13820** [pressbooks/ideas#57]
### Feature Scope

- [x] Accessibility and inclusivity
- [ ] User interface (content creation)
- [ ] User interface (network home page & catalog)
- [ ] User interface (reading)
- [x] Import/export formats
- [ ] Authoring tools
- [ ] Book themes
- [x] Integration with third-party services

### Feature Description

The ability to export text to speech might be a welcome addition for accessibility purposes. 

### Feature Use Case

Other accessibility programs like Blackboard Ally include as one of the export types a text to speech program. This would be useful for those who need accommodations, athletes on a treadmill or adult learners on their commute. 

### Other Notes

The idea that made me think of this was a solicitation email from Amazon Polly copied below. 

 
Automatically Voice Your Content with the Amazon Polly Plugin for WordPress

 
Amazon Polly is a service that allows you to convert text to speech using the same natural-sounding voice technology that powers Amazon Alexa. One sample application is the Amazon Polly plugin for WordPress which enables you to voice your website content and easily publish podcasts. This allows readers to listen to news and articles directly from the site and on-the-go.

You can also find up-to-date WordPress images and WordPress Multisite images packaged by Bitnami on the AWS Marketplace, which come preinstalled with the Amazon Polly WordPress plugin

**Comment by SteelWagstaff** [comment]
Closing in favor of https://github.com/pressbooks/ideas/issues/224
